src/cart/src/gurt/fault_inject.c

```diff
@@ -563,14 +562,7 @@ d_fault_inject_init(void)
 		D_ERROR("Failed to parse fault config file.\n");
 		D_GOTO(out, rc);
 	}
-
 	d_fault_id_mem = 0;
```

Comment:
d_fault_id_mem is not used either, so can be removed as well

Reply:
ok

Reply:
d_fault_id_mem is only really used for the lookup of d_fault_attr_mem. If you are removing this lookup, i dont see where d_fault_id_mem would be used elsewhere in code

Reply:
ok, I can remove that.